Update ZeroTrustAnalyticsPlatform to 1.1.6

vimpadawan opened the pull request:
Add view timeline script
* Add back input tag to differentiate ztap input notes
* Add script to view ZTAP timeline
* Add page to view ZTAP timeline
* Fix sync tag priority (escalate should take precidence)

<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

## Description
Notes that inbound/outbound sync can be difficult to filter in the default view. Add a view to filter this information.
Outbound tag priority was backwards (escalation should happen if selected).

## Screenshots

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

yucohen reviewed (commented):
@vimpadawan Thanks for your contribution! great work!!
It seems like you added the new script as deprecated. Was that intended? 
All the rest looks good!

vimpadawan commented:
> @vimpadawan Thanks for your contribution! great work!! It seems like you added the new script as deprecated. Was that intended? All the rest looks good!

It should not be deprecated. I have updated the script definition and added its README.
